From: Marc Kleine-Budde <mkl@pengutronix.de>
Subject: remove -frename-registers option, due to ICE on gcc-4.3.1

remove the -frename-registers options, because they ICE on gcc-4.3.1

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: procps-3.2.7/Makefile
===================================================================
--- procps-3.2.7.orig/Makefile
+++ procps-3.2.7/Makefile
@@ -141,7 +141,7 @@ ALL_CFLAGS += $(call check_gcc,-Wstrict-
 # if not debugging, enable things that could confuse gdb
 ifeq (,$(findstring -g,$(filter -g%,$(CFLAGS))))
 ALL_CFLAGS += $(call check_gcc,-fweb,)
-ALL_CFLAGS += $(call check_gcc,-frename-registers,)
+#ALL_CFLAGS += $(call check_gcc,-frename-registers,)
 ALL_CFLAGS += $(call check_gcc,-fomit-frame-pointer,)
 endif
 
